common/metrics/runtime.go

```diff
@@ -32,21 +32,22 @@ import (
 
 	"github.com/uber-go/tally"
 
+	"go.temporal.io/server/common/headers"
 	"go.temporal.io/server/common/log"
 )
 
 var (
-	// Revision is the VCS revision associated with this build. Overridden using ldflags
+	// GitRevision is the VCS revision associated with this build. Overridden using ldflags
 	// at compile time. Example:
-	// $ go build -ldflags "-X go.temporal.io/server/common/metrics.Revision=abcdef" ...
+	// $ go build -ldflags "-X go.temporal.io/server/common/metrics.GitRevision=abcdef" ...
 	// Adapted from: https://www.atatus.com/blog/golang-auto-build-versioning/
-	Revision = "unknown"
+	GitRevision = "unknown"
 
-	// Branch is the VCS branch associated with this build.
-	Branch = "unknown"
+	// GitBranch is the git branch associated with this build.
+	GitBranch = "unknown"
 
-	// Version is the version associated with this build.
-	Version = "unknown"
+	// GitVersion is the git version (tag) associated with this build.
+	GitVersion = "unknown"
```

Comment:
This version has an issue. If someone just `pull master` every time w/o fetching tags, this version becomes stale and never get updated.